src/rgw/rgw_common.cc

```diff
@@ -29,6 +31,75 @@ PerfCounters *perfcounter = NULL;
 
 const uint32_t RGWBucketInfo::NUM_SHARDS_BLIND_BUCKET(UINT32_MAX);
 
+rgw_http_errors rgw_http_s3_errors({
```

Comment:
hi, @mdw-at-linuxbox , s3 errors need "s3 code" and "s3 message" in MultiDelete request. FYI #12485 
We should add a member called error_message to rgw_http_errors struct.